Mrs_Cake/_Mrs_Cake_MrsCakeDbContextDataSet.xsd

```diff
@@ -0,0 +1,556 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
you wrote this code or come from any data insertion file? Can you please little bit elaborate?